### PR TITLE
readme: add some details of the nix CI

### PR DESCRIPTION
**Pull request by beezow**

### CODE REVIEW

File: README.mkdn [hunk 1]
```diff
@@ -296,6 +301,43 @@ before running tests.
...
+      app = "demo-rv64-qemu-virt";
+      toml = "app/demo-rv64-qemu-virt/app.toml";
+      # specify this attribute for 64bit apps
+      sixtyfour = true;
+    };
+    ...
+};
+```
+Then add the package to `checks`, and override the `doCheck` parameter.  
+Overriding this parameter will running any build time test on the app.
```
Comment (furquan-rivos) [inline README.mkdn]: will run?

**Review by furquan-rivos — DISMISSED** [review]

File: README.mkdn (hunk 1 above)
Comment (beezow) [reply README.mkdn]: good catch

File: README.mkdn [hunk 2]
```diff
@@ -59,8 +59,8 @@ The repo is laid out as follows.
 
 # Developing
 
-We currently support Linux and Windows as first-tier platforms. macOS is also
-used on a daily basis by Oxide employees, but is not tested in CI.  The build
+We currently support Linux as first-tier platforms. macOS is also
+used on a daily basis by Oxide and Rivos employees.  The build
 probably also works on Illumos; if anyone would like to step up to maintain
 support and a continuous build for Illumos or macOS, we'd love the help.
```
Comment (FawazTirmizi) [inline README.mkdn:65]: Nit: the reference to macOS could be removed here.

**Review by FawazTirmizi — APPROVED** [review]

File: README.mkdn (hunk 2 above)
Comment (beezow) [reply README.mkdn:65]: IDK, we still need people to test stuff on mac as the CI builds are not necessarily hermetic.

**Pull request merged**